pipelines/concourse.yml

```diff
@@ -426,6 +427,44 @@ jobs:
   on_success: *fixed-concourse
   on_failure: *broke-concourse
 
+- name: k8s-topgun-pks
+  public: true
+  serial: true
+  plan:
+  - in_parallel:
+    - get: concourse
+      passed: [k8s-smoke]
+      trigger: true
+      tags: [pks]
+    - get: version
+      passed: [k8s-smoke]
+      trigger: true
+      tags: [pks]
+    - get: concourse-rc-image
+      passed: [k8s-smoke]
+      trigger: true
+      tags: [pks]
+      params: {format: oci}
+    - get: unit-image
+      passed: [k8s-smoke]
+      trigger: true
+      tags: [pks]
+    - get: charts
+      trigger: true
+      passed: [k8s-smoke]
+      tags: [pks]
+    - get: ci
```

Comment:
Why aren't we using the `tags` here as well?

Reply:
Missed it there, added it now